app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  validates :name, presence: true
+  has_many :votes, dependent: :nullify
+
+  def created_at
+    attributes['created_at'].strftime("%B, %d, %Y")
```

Comment:
I think you needed to use `attributes` because you wanted to name this method `created_at` as well, but is there any other reason?